### PR TITLE
Install contrack

### PR DESCRIPTION
**Pull request by chbatey**
Refs #673

### CODE REVIEW

**Comment by chbatey** [comment]
Looks to work. I'll do the same for akka-upgrade-testing

File: scripts/setup-minikube-for-linux.sh [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 set -exu
 
+# Required by minikube
+sudo apt update
+sudo apt install conntrack
```
Comment (raboof) [inline scripts/setup-minikube-for-linux.sh:7]: doing it like https://github.com/akka/akka-sample-cluster-kubernetes-java/blob/master/.travis.yml#L8 might save some time 'update'-ing, but this is :+1: as well

Comment (chbatey) [reply scripts/setup-minikube-for-linux.sh:7]: Hrmm we already have that in our travis and doesn't seem to work for minikube

Comment (raboof) [reply scripts/setup-minikube-for-linux.sh:7]: Huh, so #649 worked for a while and then stopped working again? Weird.

Comment (chbatey) [reply scripts/setup-minikube-for-linux.sh:7]: Current builds are now passing so i'll close this

**Pull request closed without merging**